### ISSUE TITLE
Feature Request

### ISSUE THREAD

**Issue by Harsha509** [angular/protractor#3183]
Feature Request
---

Can we have  **by.exactOptions**  similar to **by.exactRepeater**  in protractor....

**Comment by NickTomlin** [comment]
I think this makes sense. Could you provide a usage example as well, just to make sure I understand?

**Comment by Harsha509** [comment]
Hi @NickTomlin,
I have a form which contains multiple ng-option tags in a list sequence. So thought of having  **exactOptions** similar to **exactRepeater** so that i can select an option exactly by its name.

**Comment by Droogans** [comment]
Could this be generalized somehow? If many of the selector strategies that accept a string instead supported regular expressions, they could be as vague or specific as a user might need (see https://github.com/angular/protractor/issues/1360).

**Comment by juliemr** [comment]
Agreed, I'm a bit worried about the # of locators exploding if everything has an exact and non-exact form.

**Comment by juliemr** [comment]
This would be a breaking change though (unless we renamed locators), so probably in the icebox for now.
